### PR TITLE
Ramone/service providers

### PR DESCRIPTION
**Pull request by Ramone-HL**
Added Service Providers section

### CODE REVIEW

File: docs/horizen_eon/06-service_providers/eon_service_providers.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Flair provides real-time and historical custom data indexing for EON. It offers reusable indexing primitives (such as fault-tolerant RPC ingestors, custom processors, re-org aware database integrations) to make it easy to receive, transform, store and access your on-chain data.
+
+See the [Flair](https://github.com/HorizenOfficial/evm-documentation/pull/78) documentation for more information. 
+
+
+## The Graph Protocol
+
+The Graph is an indexing protocol for organizing blockchain data and accessible with GraphQL.
+
+Request for [EON Graph-specific](http://info@horizen.io) documentation.
```
Comment (rabyyuson) [inline docs/horizen_eon/06-service_providers/eon_service_providers.md:51]: Are we trying to achieve this?

```suggestion
Request for [EON Graph-specific](mailto:info@horizen.io) documentation.
```

File: docs/horizen_eon/06-service_providers/eon_service_providers.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# Service Providers: Technology Features
+
+As new partners and integrations are on-boarding to the EON ecosystem, the need to use the following service providers’ technologies is imperative for a successful EVM-compatible sidechain:
+
+
+## Ankr  
+
+Ankr provides a Web3 infrastructure that has many use cases, from Web3 API, staking, blockchain deployment tool, and smart contract execution. 
+
+Request for [EON Ankr-specific](http://info@horizen.io) documentation.
```
Comment (rabyyuson) [inline docs/horizen_eon/06-service_providers/eon_service_providers.md:19]: I think we are trying to make this an email link. If so, it should be like the following:

```suggestion
Request for [EON Ankr-specific](mailto:info@horizen.io) documentation. 
```

File: docs/horizen_eon/06-service_providers/eon_service_providers.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+Developers use Covalent to build multi-chain applications like crypto wallets, NFT galleries, and investor dashboard tools utilizing data from various blockchain networks. 
+
+See the [Covalent](https://www.covalenthq.com/docs/networks/horizen/) documentation for more information. 
+
+
+## Flair
+
+Flair provides real-time and historical custom data indexing for EON. It offers reusable indexing primitives (such as fault-tolerant RPC ingestors, custom processors, re-org aware database integrations) to make it easy to receive, transform, store and access your on-chain data.
+
+See the [Flair](https://github.com/HorizenOfficial/evm-documentation/pull/78) documentation for more information.
```
[the remark this answers is not in the document]
Comment (Ramone-HL) [reply docs/horizen_eon/06-service_providers/eon_service_providers.md:44]: yes, this is the correct jump/link

**Review by rabyyuson — APPROVED** [review]

File: docs/horizen_eon/06-service_providers/eon_service_providers.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+---
+title: EON-compatible Service Providers
+---
+
+The EON ecosystem is perpetually changing with new partners and integrations. To accommodate the compatibility of EON platform with third-party integration, several service provider protocols are used to facilitate the on-boarding of new dApps, services, tools, and more.
```
Comment (taijonsmith) [inline docs/horizen_eon/06-service_providers/eon_service_providers.md:5]:
```suggestion
The EON ecosystem is perpetually changing with new partners and integrations. To accommodate the compatibility of EON platform with third-party integration, several service provider protocols are used to facilitate the onboarding of new dApps, services, tools, and more.
```

File: docs/horizen_eon/06-service_providers/eon_service_providers.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+See the [EON Ecosystem](https://eon.horizen.io/ecosystem) for a description of current service providers.    
+
+
+# Service Providers: Technology Features
+
+As new partners and integrations are on-boarding to the EON ecosystem, the need to use the following service providers’ technologies is imperative for a successful EVM-compatible sidechain:
```
Comment (taijonsmith) [inline docs/horizen_eon/06-service_providers/eon_service_providers.md:12]:
```suggestion
As new partners and integrations are onboarding to the EON ecosystem, the need to use the following service providers’ technologies is imperative for a successful EVM-compatible sidechain:
```

File: docs/horizen_eon/06-service_providers/eon_service_providers.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+See the [Pyth](https://docs.pyth.network/documentation/pythnet-price-feeds/evm) documentation for more information.
+
+
+## Relink Protocol/Chainlink VRF Proxy
+
+Relink has developed a proxy service solution (Chainlink VRF Proxy) that forwards the random data request to an officially Chainlink supported chain, where it processes the result accordingly.
+
+ 
+
+A Relink oracle network signs the data generated by Chainlink and passes the original data along with the signatures to the callback method. Signature checks in the consumer base contract ensure that the data cannot be altered in transit from the relink backend.
```
Comment (taijonsmith) [inline docs/horizen_eon/06-service_providers/eon_service_providers.md:67]:
```suggestion
A Relink oracle network signs the data generated by Chainlink and passes the original data along with the signatures to the callback method. Signature checks in the consumer base contract ensure that the data cannot be altered in transit from the Relink backend.
```

File: docs/horizen_eon/09-security_guides/01-security_user.md [hunk 7]
```diff
@@ -1,111 +1,111 @@
...
+
+Cryptocurrency wallets, much like traditional wallets, require careful handling to prevent loss or theft of assets. One of the first steps to securing your crypto-wallet (Cobalt wallet, MetaMask) is by setting a seed phrase that is then stored securely with proper best practices. It's also important to regularly update your wallet software to the latest version, as updates often include important security enhancements. Be cautious of phishing attempts, where scammers might try to trick you into giving away your wallet seed phrase. Always check the website address when accessing your online wallet and never click on suspicious links. Lastly, consider using hardware wallets, especially for storing large amounts of cryptocurrency. These wallets store your private keys offline, keeping them safe from online threats.
+
+There are no official user support centers for wallets – any links you may see that claim to be able to provide support are looking to extract sensitive information like your seed phrase. 
+
+However, there are a lot of good articles on Wallet Safety that provide best practices and advice. We have linked to some of the prominent ones below:
```
Comment (taijonsmith) [inline docs/horizen_eon/09-security_guides/01-security_user.md:56]:
```suggestion
However, there are a lot of good articles on wallet safety that provide best practices and advice. We have linked to some of the prominent ones below:
```

**Comment by taijonsmith** [comment]
Just some small things I noticed, once fixed I'll approve

**Review by taijonsmith — APPROVED** [review]

**Pull request merged**